<launch>
  <node pkg="position_tracker" type="multi_object_detector" name="multi_object_detector" output="screen">
    <param name="object_name" value="Ball" />
</node>
</launch>
